docs/src/pages/components/data-grid/rendering/rendering.md

```diff
@@ -213,7 +213,7 @@ You can change the density of the rows and the column header.
 
 ### Density prop
```

Comment:
How about we document how to display the density  toolbar first?

Reply:
I've reorder them